### PR TITLE
Added German Translation

### PR DESCRIPTION
**Pull request by data-niklas**
Great App btw.

### CODE REVIEW

**Comment by data-niklas** [comment]
Added, that songs (library) can now be sorted according to the duration of each song (asc and desc).
Sorting according to the date, when the file was added would be nice, but the date is not stored in the MusicModel. I did not want to change the MusicModel, so I only added the duration sorting for now.

Best regards

**Comment by data-niklas** [comment]
Now allows sliding between the main fragments (home, library, albums, artists, playlists). Does not use the entry and exit animation any more. Now uses the default ViewPager2 sliding animation.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-    <p align="center">
+<p align="center">
```
Comment (HardcodeCoder) [inline README.md]: Just a pointer, avoid making unnecessary commits

**Comment by HardcodeCoder** [comment]
Also, Please consider cleaning up unnecessary imports. (If any) (If using Android studio reformat your code)

**Review by HardcodeCoder — COMMENTED** [review]
The implementation is good but it kinda interferes with the home section where we have `horizontal recycler view`. When we try to scroll the horizontal list in home section the ViewPager get's triggered and moves to the next section. Try disabling the swipe gesture so that navigation is possible only via `navigation bar`

**Comment by data-niklas** [comment]
I could only disable the swipe gesture for the home screen, or just remove the changes to the old code.
I'll also translate your changes. The one commit above `<p align="center">` was necessary, because Github does not preview the README correctly, if the tag is indented, and it was somehow changed in the first commit. I'll clean up all of the commits and commit them again, so that there are only necessary commits.

**Comment by HardcodeCoder** [comment]
That's great. Revert the fragment transition changes as of now.  I will add a option to either use the default navigation style or the view pager style. Then we can merge back those changes.

**Comment by HardcodeCoder** [comment]
Let me know once changes are ready to be merged.

**Comment by data-niklas** [comment]
My git commit history isn't the prettiest, but all of the changes are ready to be merged. I reverted the ViewPager2, so now sliding isn't possible anymore. The translation has also been added and the sorting according to the duration still works. Code has been reformatted.

**Comment by HardcodeCoder** [comment]
No issues, I will merge them asap.

**Comment by HardcodeCoder** [comment]
I might be bothering you too much, but could you please change the strings from `Longest to shortest `to `Duration Desc` and `Shortest to longest` to `Duration Asc`. I think it's more clearer this way. And thank you for your contribution.

**Comment by data-niklas** [comment]
No problem, I'll change them ASAP

**Comment by data-niklas** [comment]
Are there icons for arrows pointing upwards and downwards? We could use those arrows instead of asc. and desc.

**Comment by HardcodeCoder** [comment]
> Are there icons for arrows pointing upwards and downwards? We could use those arrows instead of asc. and desc.

No there aren't any.

**Comment by data-niklas** [comment]
I am now using ↓  and ↑ instead of desc and asc and it seems to work

**Comment by HardcodeCoder** [comment]
> I am now using ↓ and ↑ instead of desc and asc and it seems to work

Great that's better

**Pull request merged**